core/templates/dev/head/pages/admin/admin_navbar_directive.html

```diff
@@ -1,14 +1,16 @@
 <script type="text/ng-template" id="admin/navbar">
   <nav class="navbar navbar-default oppia-navbar oppia-prevent-selection" role="navigation">
-    <div class="navbar-container" style="background-color: #00376d">
+    <!-- IMPORTANT: The change to the navbar color should be reverted prior to merge into develop. -->
```

Comment:
Leaving a comment on this file to flag the "IMPORTANT" bits here -- please resolve it just prior to merge.